### PR TITLE
TurboTax has 2FA.

### PR DESCRIPTION
**Pull request by mikeatlas**

### CODE REVIEW

**Comment by mikeatlas** [comment]
> ```No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.
The build has been terminated```

**Comment by jamcat22** [comment]
It's ok. Travis has failing lately.

**Comment by mikeatlas** [comment]
re-running, hoping for a 2nd chance with Travis build

**Comment by Carlgo11** [comment]
Travis always fails. We're looking into it.

**Comment by Carlgo11** [comment]
So there's no real way for you to fix it atm. Just leave travis be

File: _data/finance.yml [hunk 1]
```diff
@@ -39,7 +39,11 @@ websites:
       url: https://turbotax.intuit.com/
       twitter: turbotax
       img: turbotax.png
-      tfa: No
+      tfa: Yes
+      phone: No
```
Comment (Carlgo11) [inline _data/finance.yml]: If phone is false then set just remove the line.

**Comment by mikeatlas** [comment]
@Carlgo11 `-phone: no` removed.

File: _data/finance.yml [hunk 2]
```diff
@@ -39,7 +39,10 @@ websites:
       url: https://turbotax.intuit.com/
       twitter: turbotax
       img: turbotax.png
-      tfa: No
+      tfa: Yes
+      sms: Yes
+      software: yes
```
Comment (Carlgo11) [inline _data/finance.yml:44]: `yes` > `Yes`

**Comment by mikeatlas** [comment]
I regret trying to help out.

**Comment by mikeatlas** [comment]
I've seen your team fix "volunteered" PRs in the past before, instead of just nitpicking submissions. What a waste of time. Example: #894 Good day.

**Comment by mikeatlas** [comment]
https://ttlc.intuit.com/questions/2903072 is all someone else needs to recreate this PR.

**Comment by jamcat22** [comment]
@mikeatlas I'm apologize about this PR too.

I'll add it to the backlog. I'll also make sure that I explain the PR process better in our docs (particularly with the `Twitter:` and `[method]: No` lines, which are a bit confusing right now).

If you ever feel like contributing again, feel free to submit a PR or an issue if you don't want to make a branch.

(Also read my comments on #1310 for more info.)

**Pull request closed without merging**